sky/serve/load_balancer.py

```diff
@@ -27,7 +27,9 @@ class SkyServeLoadBalancer:
     policy.
     """
 
-    def __init__(self, controller_url: str, load_balancer_port: int) -> None:
+    def __init__(self, controller_url: str, load_balancer_port: int,
+                 ssl_keyfile: Optional[str],
+                 ssl_certfile: Optional[str]) -> None:
```

Comment:
It seems `ssl_keyfile` and `ssl_certfile` should be specified together. Having two arguments might be a bit verbose. How about we create a dataclass somewhere and change the argument type `Optional[SSLCredentials]`
```python
@dataclassess.dataclass
class SSLCredential:
    key_path: str
    cert_path: str
```